tests/v2/validation/Dockerfile.registries

```diff
@@ -0,0 +1,26 @@
+FROM registry.suse.com/bci/golang:1.19
+
+# Configure Go
+ENV GOPATH /root
+ENV PATH ${PATH}:/root/bin
+
+ENV WORKSPACE ${GOPATH}/src/github.com/rancher/rancher
+ENV WORKSPACE2 ${GOPATH}/src/github.com/rancherlabs/corral-packages
+
+WORKDIR $WORKSPACE
+
+COPY ["./rancher", "$WORKSPACE"]
+COPY ["./corral-packages", "$WORKSPACE2"]
+
+ENV CORRAL_VERSION="v1.1.1"
+
+RUN mkdir -p /root/.ssh
+RUN cp ${WORKSPACE}/tests/v2/validation/.ssh/jenkins-* /root/.ssh
```

Comment:
Doing this step is unnecessary you just need to set the ssh path in the corral config, and corral will know to use the correct ssh key.

Reply:
I also think we should rename this Dockerfile and the Jenkins with corralsetup suffix, I can totally use these files for my air gap work.

Reply:
> Doing this step is unnecessary you just need to set the ssh path in the corral config, and corral will know to use the correct ssh key

I tried this, but corral I think keeps using the corral generated ssh key

Reply:
Really? I tired this on my local machine and was able to get it to work. Are you sure you have the sshPath correct?

Reply:
Nevermind you 100% ignore this comment.